### PR TITLE
Cleanup migration script and assets, update README (AO-13011)

### PR DESCRIPTION
**Pull request by lootek**
Beta version

JIRA: AO-13011

JIRA: AO-13028

JIRA: AO-13030
JIRA: AO-13031
JIRA: AO-13032
JIRA: AO-13033
JIRA: AO-13034
JIRA: AO-13035

### CODE REVIEW

File: lib/scout/client.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    def scout_configuration
+      @scout_configuration ||= YAML.load(File.read(config_file))
+    end
+
+    def fetch_client_roles
+      roles = {}
+
+      response = api_client.make_request("/api/v2/account/clients/roles")
+      response.reject! {|r| r["name"] == "All Servers"}
+      response.map! {|r| r["name"].gsub(/(\s+|\W+)/, "_")}
```
Comment (okapusta) [inline lib/scout/client.rb]: We've changed a bit how roles are mapped to tags in AO. Now it's: role => true

File: lib/scout/rest.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      def development?
+        ENV["RACK_ENV"] == "development"
+      end
+
+      def staging?
+        ENV["RACK_ENV"] == "staging"
+      end
+    end
+
+    API_HOST = if development?
```
Comment (okapusta) [inline lib/scout/rest.rb]: Should we keep this? Ultimately we will just want to use prod servers for migrating customers

File: lib/scout/plugins.rb [hunk 3]
```diff
@@ -63,19 +63,19 @@ def save
 
   private
 
-  def save_ruby_code
-    save_file("#{name}.rb", code)
-  end
+    def save_ruby_code
```
Comment (okapusta) [inline lib/scout/plugins.rb]: Not a fan of indenting private methods tbh. I used to do that but not anymore. I leave the decision up to you https://github.com/rubocop-hq/ruby-style-guide#indent-publicprivateprotected

File: lib/scout/scout.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
+require_relative "client"
+require_relative "plugins"
+
+require "fileutils"
+
+class Scout
+  class << self
+    def deactivate
+      system "scoutctl stop"
+      system "mv -f /var/lib/scoutd/client_history.yaml /var/lib/scoutd/client_history.yaml.bak"
```
Comment (okapusta) [inline lib/scout/scout.rb]: Consider moving history file path to constant

File: templates/statsd-task.yaml.erb [hunk 5]
```diff
@@ -16,15 +16,15 @@ workflow:
         bridge_use_json_fields: true
     tags:
       /:
-        <% if roles %>
+          <% if roles %>
```
Comment (okapusta) [inline templates/statsd-task.yaml.erb]: Unnecessary indent? Makes file less readable IMO

Comment (lootek) [reply templates/statsd-task.yaml.erb]: tbh I don't like this indent to but this is how RubyMine did it... :?

File: lib/scout/rest.rb (hunk 2 above)
Comment (lootek) [reply lib/scout/rest.rb]: removed

File: lib/scout/plugins.rb (hunk 3 above)
Comment (lootek) [reply lib/scout/plugins.rb]: fixed

File: lib/scout/scout.rb (hunk 4 above)
Comment (lootek) [reply lib/scout/scout.rb]: done

**Pull request merged**